src/components/ButtonGroup.js

```diff
@@ -0,0 +1,101 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Button = require('./Button');
+
+const StyleConstants = require('../constants/Style');
+
+const ButtonGroup = React.createClass({
+  propTypes: {
+    buttons: React.PropTypes.arrayOf(React.PropTypes.shape({
+      icon: React.PropTypes.string,
+      text: React.PropTypes.string
+    }).isRequired),
+    controlArrows: React.PropTypes.bool,
```

Comment:
I'm thinking we don't provide a prop or any type of specific support for these arrows. If a consumer of this component needs arrow buttons, they can just pass them in through the `buttons` prop.